apps/web/e2e/beta/github-revoke.spec.ts

```
@@ -0,0 +1,356 @@
+        .catch(() => undefined);
+      if (disconnected) {
+        try {
+          await restoreGithubConnection(page);
+        } catch (error) {
+          // Surface the restore failure without hiding the primary assertion error.
+          await test.info().attach("github-restore-failure", {
+            body: String(error),
+            contentType: "text/plain",
+          });
+          if (!test.info().errors.length) {
+            throw error;
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)